From f2f4b8f16cb19c346e5f742471fc97806cf8c142 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 20 Mar 2013 16:20:18 +0800
Subject: [PATCH 178/602] ENGR00255113 update fsl-sdcard-partition.sh to
 enlarge the cache partition size

enlarge cache partition size to be 512MB, which previous size is not enough for
OTA function

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 common/tools/fsl-sdcard-partition.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/tools/fsl-sdcard-partition.sh b/common/tools/fsl-sdcard-partition.sh
index df19bf5..ff8185c 100755
--- a/common/tools/fsl-sdcard-partition.sh
+++ b/common/tools/fsl-sdcard-partition.sh
@@ -4,7 +4,7 @@
 BOOTLOAD_RESERVE=8
 BOOT_ROM_SIZE=8
 SYSTEM_ROM_SIZE=512
-CACHE_SIZE=256
+CACHE_SIZE=512
 RECOVERY_ROM_SIZE=8
 VENDER_SIZE=8
 MISC_SIZE=8
-- 
1.8.0

